examples/advanced/flutter/pubspec.lock

```diff
@@ -0,0 +1,188 @@
+# Generated by pub
```

Comment:
Although this is in the link below as something to "not commit", my guess is that this file IS something that makes everything in the example run correctly. Can you verify this file is required for the example? Thanks!

Further info on what to commit/not commit here:
https://dart.dev/guides/libraries/private-files

Reply:
The .pubspec.lock is generated on pub get, so it's safe to delete